App.tsx

```diff
@@ -1,10 +1,34 @@
+import { useContext } from 'react';
 import { SafeAreaView, Text } from 'react-native';
+import { SafeAreaProvider } from 'react-native-safe-area-context';
+
+import SettingsContext, { SettingsContextProvider } from '@config/SettingsContext';
+import PlatformDependentStatusBar from '@config/platformDependentStatusBar';
+
+import GlobalThemedStyles from '@themes/globalStyles';
 
 const App = () => {
   return (
-    <SafeAreaView>
-      <Text>Tuneify</Text>
-    </SafeAreaView>
+    <SettingsContextProvider>
+      <SafeAreaProvider>
+        <Main />
+      </SafeAreaProvider>
+    </SettingsContextProvider>
+  );
+};
+
+const Main = () => {
+  const { theme } = useContext(SettingsContext);
+
+  const globalStyles = GlobalThemedStyles(theme);
+
+  return (
+    <>
+      <PlatformDependentStatusBar />
+      <SafeAreaView style={[globalStyles.screen]}>
```

Comment:
why are there two instances of "SafeArea"?

Reply:
If it only solving status bar issue, a more meaningful name to the component can be used